libs/rudder-integration-braze-react-native/project.json

```diff
@@ -6,7 +6,7 @@
   "tags": [],
   "targets": {
     "build": {
-      "executor": "@nrwl/web:rollup",
+      "executor": "@nx/rollup:rollup",
```

Comment:
is that correct? should we not use https://nx.dev/nx-api/web here?

Reply:
After updating the `nx` to the latest version, `@nrwl/web:rollup` is no longer working, due to that `npm run build` command got failed. So, after some research, I used `@nx/rollup:rollup` and now the build is successful. Hence, I think, it is correct.

Coming to the `@nx/web`: Although I don't have proper knowledge about it, I tried and it doesn't seem to work and the build is failing. (I replaced `@nx/rollup:rollup` with `@nx/web` and `@nx/web:rollup` and build failed).

Moreover, we need to use `rollup`, which previously came bundle inside `@nrwl/web`-`executor.json` file. Whereas, in the latest update, that is missing.

If would like me to try with something specific, please let me know.

Reply:
I would help to know what the build error was. Now i have to checkout and try to build my self to figure it out

Reply:
anyow seems like the latest version of web only has one executor the "file-server" so at some pont there was a breaking change. rollup is good to use for this case

Reply:
Sure. Let me know once you find an alternative solution for this.
Meanwhile, you can also take a look on the error, I'm getting:
<img width="930" alt="image" src="https://github.com/rudderlabs/rudder-sdk-react-native/assets/64667840/510b883f-251d-4548-821c-b612510da430">